usagov/usagov-2021
# USAGOV-1134-Reduce-Shift: Update _uswds-theme-custom-styles.scss

## Jira Task

https://cm-jira.usa.gov/browse/USAGOV-1134

## Description

Nicole saw " after loading the page, the first click jumps, then subsequent clicks do not. This happens in both environments, and also even if I close the page and come back to it again."

I think because the height of the image wasn't set the nav-bar header isn't set the first time it loads when the image isn't cached so I give the surrounding image div a height and width to make it not do the jump on the first click. It looks like to me that even though we use max-width for the logo when on desktop the logo is always 5rem by 5rem. 

The other things I was considering for this was that a script was affecting it but turning of JS in Dev Tools it's still happening or that it would be cause by the float on the phone number. Let me know if you think there's a better way to approach this. 

## Type of Changes

  - Add screenshot showing what it should look like

## Testing Instructions



### Requires New Config

### Requires New Content

### Validation Steps
- On localhost toggle to Spanish
- See if the phone number jumps (read the jira ticket for more context) 

## Security Review




## Reviewer Reminders
- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions
Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
2. Find the commit of this pull request.
3. Build and deploy the changes.
4. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
